Wikipedia/Code/WMFSearchViewController.m

```diff
@@ -319,30 +319,8 @@ - (void)textFieldDidBeginEditing:(UITextField *)textField {
 - (IBAction)textFieldDidChange {
     NSString *query = self.searchField.text;
 
-    dispatchOnMainQueueAfterDelayInSeconds(0.4, ^{
```

Comment:
Ok to remove because when we get results we check if the results are for the search term (the success block captures the original search term):  
https://github.com/wikimedia/wikipedia-ios/pull/2298/files#diff-6b0e15d84fb42eb1eea4b6ee96fe8b3bR419